charts/flyte-core/values.yaml

```diff
@@ -279,6 +279,8 @@ flyteagent:
         defaultAgent:
           endpoint: "dns:///flyteagent.flyte.svc.cluster.local:8000"
           insecure: true
+  # -- Labels for flyteagent pods
+  podLabels: {}
```

Comment:
cc @ddl-ebrown  I remove `podLabels` from the helper in this [PR](https://github.com/flyteorg/flyte/pull/4874/files#diff-e2eb39162dcb726f53d87b4c2ef9d5af3b2450ac5f30508deef6bf0cded88a2bL105-L118).

Also, would you mind taking a look at the new agent API? We want to make it as simple as possible, so make some change to it. https://github.com/flyteorg/flytekit/pull/2146

Reply:
> just as a workaround for the chart install issue
yes

> I added it recently in https://github.com/flyteorg/flyte/pull/4756)

I see, got it. Let's merge your PR first.